wled00/FX.h

```diff
@@ -571,7 +572,7 @@ typedef struct Segment {
     uint8_t  currentMode(void);                 // currently active effect/mode (while in transition)
     uint32_t currentColor(uint8_t slot);        // currently active segment color (blended while in transition)
     CRGBPalette16 &loadPalette(CRGBPalette16 &tgt, uint8_t pal);
-    CRGBPalette16 &currentPalette(CRGBPalette16 &tgt, uint8_t paletteID);
+    inline CRGBPalette16 &currentPalette(uint8_t paletteID);
```

Comment:
Inlining will increase code size which is unacceptable as we are already at 98.5% flash size.